caravel/assets/javascripts/SqlLab/reducers.js

```diff
@@ -13,13 +13,39 @@ const defaultQueryEditor = {
 
 export const initialState = {
   alerts: [],
-  queries: [],
+  queries: {},
   queryEditors: [defaultQueryEditor],
   tabHistory: [defaultQueryEditor.id],
   tables: [],
   workspaceQueries: [],
 };
 
+function removeFromDict(state, arrKey, obj, idKey = 'id') {
```

Comment:
i think we could rename `removeFromDict` to `removeQueries`, and below `alterQueries` and addQueries`. makes more sense to call it what it what it is storing rather than the format it is being stored in, dictionary/object. also, in javascript we call dictionaries objects or maps. 

what do you think about changing the naming format here?